src/main/jbake/content/jms-examples.adoc

```diff
@@ -1,30 +1,30 @@
 type=page
 status=published
-title=Java Message Service Examples
+title=Jakarta Messaging Examples
 next=jms-examples001.html
 prev=jms-concepts006.html
 ~~~~~~
-Java Message Service Examples
+Java Messaging Examples
 =============================
 
 [[BNCGV]][[java-message-service-examples]]
```

Comment:
Ideally any of these tags would be updated as well, e.g., to "jakarta-message-service-examples".